clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py

```diff
@@ -29,7 +29,24 @@ class Dashboard(g.Dashboard):
         default=g.Templating(
             list=[
                 # Make it possible to use $source as a source.
-                g.Template(name="source", type="datasource", query="prometheus")
+                g.Template(name="source", type="datasource", query="prometheus"),
+                g.Template(
```

Comment:
I think this adds variables to all dashboards including the ones which has nothing to do with etcd (e.g. dns, network). Is this correct?

Can we somehow avoid that?

Reply:
Done.